bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -115,8 +121,8 @@ def add_or_update_advisory(data, html):
     kwargs = {
         'id': mfsa_id,
         'title': data.pop('title'),
-        'impact': data.pop('impact'),
-        'reporter': data.pop('reporter', None),
+        'impact': data.pop('impact', None) or '',
```

Comment:
@glogiotatidis this basically means that default is `''`, so I think it doesn't matter much in practice.